### PR TITLE
Extended Analytics.Api for Seller.Web

### PR DESCRIPTION
**Pull request by SebaKas21**

### CODE REVIEW

File: be/src/Project/Services/Analytics/Analytics.Api/Services/SalesAnalytics/SalesService.cs [hunk 1]
```diff
@@ -52,31 +52,47 @@ public async Task CreateAsync(CreateSalesAnalyticsServiceModel model)
...
                     await _context.ClientDimensions.AddAsync(clientDimension.FillCommonProperties());
+                } 
+                else
+                {
+                    clientDimension.Name = client.Name;
+                    clientDimension.OrganisationId = client.OrganisationId;
                 }
 
-                var locationDimension = await _context.LocationDimensions.FirstOrDefaultAsync(x => x.CountryId == client.CountryId);
+                Nullable<Guid> locationDimensionId = null;
```
Comment (dawiddworak88) [inline be/src/Project/Services/Analytics/Analytics.Api/Services/SalesAnalytics/SalesService.cs]: **suggestion:** Guid?

**Review by dawiddworak88 — COMMENTED** [review]
Hi @SebaKas21 , could you check how much time it takes to open the page using C# StopWatch and Network tab in Chrome browser?

**Comment by SebaKas21** [comment]
Hi @dawiddworak88 

Can you describe on why check page uptime if it's just an API extension?

File: be/src/Project/Services/Analytics/Analytics.Api/Services/SalesAnalytics/SalesService.cs (hunk 1 above)
Comment (SebaKas21) [reply be/src/Project/Services/Analytics/Analytics.Api/Services/SalesAnalytics/SalesService.cs]: I will think about it

**Review by dawiddworak88 — APPROVED** [review]

**Pull request merged**